S2VX.Game/Editor/ToolState/SelectToolState.cs

```diff
@@ -212,7 +217,35 @@ public class SelectToolState : S2VXToolState {
             }
         }
 
-        public override void OnToolDragEnd(DragEndEvent _) => ToDrag = SelectToolDragState.None;
+        public override void OnToolDragEnd(DragEndEvent _) {
```

Comment:
Use _ for variable name only if the variable is not used, see: https://docs.microsoft.com/en-us/dotnet/csharp/discards

It's general naming convention to use _ when a parameter is necessary for some convention function, but don't use the variable. Sometimes you may see it with a fuller name to give some context, e.g. `OnToolDragEnd(DragEndEvent _event)`.

Reply:
Oh I forgot about that, Changed